src/mantine-core/src/components/Text/Text.tsx

```diff
@@ -15,8 +15,8 @@ export interface SharedTextProps extends DefaultProps {
   /** Predefined font-size from theme.fontSizes */
   size?: MantineSize;
 
-  /** Text color from theme */
-  color?: MantineColor;
+  /** Text color from theme or dimmed variant */
+  color?: 'dimmed' | MantineColor;
```

Comment:
If 'dimmed' is supposed to use in some other components too, why not simply create a utility type like below
```typescript
type WithDimmed<T> = T | 'dimmed';
```

And after use it like:
```suggestion
  color?: WithDimmed<MantineColor>;
```